setup.py

```diff
@@ -21,7 +21,7 @@
 import setuptools
 from setuptools.command import build_py
 
-VERSION = '2.2.0'
+VERSION = '2.1.1'
```

Comment:
revert please (and commit with `git commit --amend --no-edit` to ensure things stay squashed)